tests/tst_decode.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * This test is for decoding full grib2 message.
+ */
```

Comment:
Please add your name so we know who wrote this test.

Reply:
We do not need list ever single file change in the comments. Git tracks all of that info.

Reply:
Ironically untrue. All the current authorship is only understood because of comments in the code, since the code was just imported into GitHub by people other than the author.

On NCEPLIBS and UFS_UTILS we are keeping track of all authors for:
* every file (including test and build files).
* every module
* every subprogram.
These are part of the new documentation standards for NCEPLIBS. Please follow these guidelines for NCEPLIBS code changes.